src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -983,15 +991,16 @@ public void SetCookies(Uri uri, string cookieHeader)
     }
 
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     internal struct PathList
```

Comment:
Now that this is going back to being stored in `Hashtable` (and therefore is going to be boxed anyway), should it be changed back to a class?